Zend/zend_string.h

```diff
@@ -35,41 +36,82 @@ void zend_interned_strings_dtor(void);
 
 END_EXTERN_C()
 
+/* No need for a function as ZSTR_VAL() is a pure getter */
+
+#define ZSTR_VAL(zstr) (zstr)->val
+
+/* Get len */
+
+#define ZSTR_LEN(zstr) zend_string_get_len(zstr)
```

Comment:
ZSTR_VAL(), ZSTR_LEN() are a bit inconsistent with the rest of API naming, but I can't propose a better choice. May be we should think about changing all zend_string_...() into ZSTR_...(). This is very questionable...